guides/source/asset_pipeline.md

```diff
@@ -868,218 +779,384 @@ each asset will have a different file name when it is changed. This way you
-
-NOTE: `cssbundling-rails` relies on Node to process the CSS. The `dartsass-rails` and `tailwindcss-rails` gems use standalone versions of Tailwind CSS and Dart Sass, meaning no Node dependency. If you are using `importmap-rails` to handle your JavaScripts and `dartsass-rails` or `tailwindcss-rails` for CSS you could completely avoid the Node dependency resulting in a less complex solution.
-
-### dartsass-rails
-
-If you want to use [Sass](https://sass-lang.com/) in your application, [`dartsass-rails`](https://github.com/rails/dartsass-rails) comes as a replacement for the legacy `sassc-rails` gem. `dartsass-rails` uses the Dart Sass implementation in favour of deprecated in 2020 [LibSass](https://sass-lang.com/blog/libsass-is-deprecated) used by `sassc-rails`.
-
-Unlike `sassc-rails` the new gem is not directly integrated with Sprockets. Please refer to the [gem homepage](https://github.com/rails/dartsass-rails) for installation/migration instructions.
-
-WARNING: The popular `sassc-rails` gem is unmaintained since 2019.
-
-### tailwindcss-rails
-
-[`tailwindcss-rails`](https://github.com/rails/tailwindcss-rails) is a wrapper gem for [the standalone executable version](https://tailwindcss.com/blog/standalone-cli) of Tailwind CSS v3 framework. Used for new applications when `--css tailwind` is provided to the `rails new` command. Provides a `watch` process to automatically generate Tailwind output in development. For production it hooks into `assets:precompile` task.
+2. Delete the `config/assets.rb` and `assets/config/manifest.js` files from your
+   project.
+
+3. If you've already upgraded to Rails 8, then Propshaft is already included in
+   your application. Otherwise, install it using `bundle add propshaft`.
+
+4. Remove the `config.assets.paths << Rails.root.join('app', 'assets')` line
+   from your `application.rb` file.
+
+5. Migrate asset helpers by replacing all instances of asset helpers (e.g.,
+   `image_url`) with standard URLs because Propshaft utilizes relative paths.
+   For example, `image_url("logo.png")` will become `url("/logo.png")`.
+
+6. If you're relying on Sprockets for transpiling, you'll need to switch to a
+   Node-based transpiler like Webpack, esbuild, or Vite. You can use the
+   `jsbundling-rails` and `cssbundling-rails` gems to integrate these tools into
+   your Rails application.
+
+For more information, you can read the [detailed guide on how to migrate from
+Sprockets to
+Propshaft](https://github.com/rails/propshaft/blob/main/UPGRADING.md).
+
+## Advanced Asset Management
+
+Over the years, there have been multiple default approaches for handling assets.
+As the web evolved, we began to see more JavaScript-heavy applications. In The
+Rails Doctrine we believe that [The Menu Is
+Omakase](https://rubyonrails.org/doctrine#omakase) so we focused on the default
+setup within Propshaft.
```

Comment:
```suggestion
As the web evolved, we began to see more JavaScript-heavy applications.
The Rails Doctrine posits that [The Menu Is
Omakase](https://rubyonrails.org/doctrine#omakase), so Propshaft focuses
on delivering a production-ready setup with modern browsers by default.
```

Reply:
@sedubois Thanks for your suggestion! We aim to use simple language in the guides, so "posits" feels a bit outside that scope. That said, I really liked the latter part of your suggestion and have incorporated it—appreciate the input!